### PR TITLE
Add mdbook guide for implementing Northstar clients

### PR DESCRIPTION
**Pull request by alfonsoros88**

### CODE REVIEW

File: .github/workflows/mdbook.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - guide
```
Comment (alfonsoros88) [inline .github/workflows/mdbook.yml:6]: This has to be changed to master once the pr is ready to be merged

File: guide/src/client/client.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Implementing a client
+
+This tutorial demonstrates how to interact with **Northstar** using its **API**.
+
+**Northstar** uses **JSON** to encode the messages shared with clients. This is
+a common approach that facilitate clients to be implemented in any programming
+language.  However, **Northstar** as a library, provides a convenient `Client`
```
Comment (flxo) [inline guide/src/client/client.md:7]: We should generate `rustdoc` for the `api` module ;-)

File: guide/src/client/repositories.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Northstar Repositories
+
+Repositories are used to group sets of containers. These are specified in
+the Northstar configuration (see `northstar.toml`).
+
+Containers can be added or removed from repositories at runtime by
+**installing** and **uninstalling** them. Note that repositories will not allow
+these actions if they are configured with the `writable` parameter set to
+`false`.
```
Comment (flxo) [inline guide/src/client/repositories.md:9]: I think we do not have this option.

**Review by flxo — COMMENTED** [review]
@alfonsoros88  Nice!
 @pclesr You're a walking grammar checker. Can you please have a look?

File: guide/src/client/repositories.md (hunk 3 above)
Comment (alfonsoros88) [reply guide/src/client/repositories.md:9]: You are right, funny that this doc is already outdated :')

**Pull request merged**